src/seedu/addressbook/data/AddressBook.java

```diff
@@ -133,4 +133,5 @@ public int hashCode() {
         // use this method for custom fields hashing instead of implementing your own
         return Objects.hash(allPersons, allTags);
     }
+    
```

Comment:
Be careful of unnecessary or accidental changes here as it can cause unnecessary merge conflict.